modules/RTC/BridgeChannel.js

```diff
@@ -412,6 +424,12 @@ export default class BridgeChannel {
         channel.onclose = event => {
             logger.info(`Channel closed by ${this._closedFromClient ? 'client' : 'server'}`);
 
+            if (channel !== this._channel) {
+                logger.info('Skip close handler, channel instance is not equal to stored one');
```

Comment:
Can you please make this debug instead of info?

Reply:
done!